Competition2017/src/competition/subsystems/shooter_wheel/ShooterWheelSubsystem.java

```diff
@@ -98,7 +102,18 @@ public void setTargetSpeedForRange(double rangeInInches) {
     }
     
     public boolean isWheelAtSpeed() {
-        return super.getSpeed() >= super.getTargetSpeed() * wheelSpeedThresholdPercentage.get();
+        // Speed is in native units and can be negative
+        double currentSpeed = Math.abs(super.getSpeed());
+        if (currentSpeed <= 1) {
```

Comment:
The agitator was firing when we didn't have that if statement. It was a temporary fix but once we have time we should look back into the logic and fix it.